tests/atomistics/structure/test_atoms.py

```diff
@@ -14,6 +14,7 @@
 from pyiron_base import FileHDFio, ProjectHDFio, Project
 from ase.cell import Cell as ASECell
 from ase.atoms import Atoms as ASEAtoms
+from ase.geometry.geometry import find_mic
```

Comment:
I get a bit of an uneasy feeling having our test simply run our code against theirs. It's not something I'd say needs to change before the code is merged and the only easy brute force test I can think of is making sure that none of the distances exceed more than half the longest vector that can sit inside the box (e.g. corner-to-corner). 

However, I looked back in more detail at your conversation just now and I guess my unease is not unjustified. I have no suggestion atm, but I would be happier with an alternative if you can think of one.

Reply:
I don't know if the changes I just made answer what you're saying, but I anyway removed the ASE `find_mic`